layouts/shortcodes/cve-feed.html

```diff
@@ -1,5 +1,6 @@
 <table class="security-cves">
-  <caption>{{ T "cve_table" }}</caption>
+  {{ $feed := getJSON .Site.Params.cveFeedBucket }}
+  <caption>{{ T "cve_table" }} (updated on {{ $feed._kubernetes_io.updated_at | time.Format "02 Jan 2006 at 15:04:05 MST" }})</caption>
```

Comment:
Use a localized / localizable time format, not `"02 Jan 2006 at 15:04:05 MST"` (which embeds an English word)

Reply:
@mtardy can you try the options from here to resolve @sftim's suggestion: https://gohugo.io/functions/dateformat/ ?

Reply:
@mtardy another example: https://github.com/kubernetes/website/blob/3ca34b9a3be454055ba234f1d2ff7d55809b5040/layouts/index.rss.xml#L24

Reply:
Try:
```html
  <caption>{{ T "cve_table" }} (updated on {{ $feed._kubernetes_io.updated_at | time.Format ( T "cve_table_date_format" ) }})</caption>
```

Reply:
Yes, that's perfect thanks for your help!